### ISSUE TITLE
The filename, directory name, or volume label syntax is incorrect.

### ISSUE THREAD

**Issue by thomaslevesque** [adamralph/simple-exec#112]
Hey @adamralph,

After I upgraded to 5.0.0, my build scripts (I have several like [this one](https://github.com/thomaslevesque/WeakEvent/blob/master/tools/build/Build.cs)) started failing like this:

```
Bullseye/build: Starting...
cmd.exe /c "dotnet" build -c "Release" /bl:"C:\git\WeakEvent\artifacts\logs\build.binlog" "WeakEvent.sln"
The filename, directory name, or volume label syntax is incorrect.
Bullseye/build: Failed! The process exited with code 1. (36.6 ms)
Bullseye: Failed! (default) (70.7 ms)
```

If I copy/paste the command line from the output above, it works fine.

The same script works fine with previous versions of SimpleExec.

Could it be related to #100? Is there any other known breaking change?

**Comment by adamralph** [comment]
Thanks for raising this @thomaslevesque. I'll look into it ASAP.

**Comment by adamralph** [comment]
Released in [RC 1](https://www.nuget.org/packages/SimpleExec/5.0.1-rc.1).

@thomaslevesque can you please try [5.0.1 RC 1](https://www.nuget.org/packages/SimpleExec/5.0.1-rc.1)?

**Comment by thomaslevesque** [comment]
That was fast! It seems to work fine with 5.0.1 RC1.

Thanks!

**Comment by adamralph** [comment]
Great! FYI I've released 5.0.1 RTM, and tweet is scheduled for tomorow. Thanks again!
